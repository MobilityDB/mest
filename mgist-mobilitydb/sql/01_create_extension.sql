CREATE EXTENSION mgist_mobilitydb CASCADE;
