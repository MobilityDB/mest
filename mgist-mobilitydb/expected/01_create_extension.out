CREATE EXTENSION mgist_mobilitydb CASCADE;
NOTICE:  installing required extension "mobilitydb"
NOTICE:  installing required extension "postgis"
NOTICE:  installing required extension "mgist"
